kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtDhcpLease.java

```diff
@@ -0,0 +1,48 @@
+package org.eclipse.kura.web.shared.model;
```

Comment:
Please add the copyright header also here

Reply:
OK.